src/main/resources/otj-pmd-rulesets/ot-pmd.xml

```diff
@@ -71,6 +71,8 @@
         <exclude name="AvoidPrefixingMethodParameters" />
         <exclude name="ShortClassName" />
         <exclude name="GenericsNaming" />
+        <!-- Way too picky @Goofy("star") vs @Goofy(value = "star") is ok -->
+        <exclude name="UnnecessaryAnnotationValueElement" />
```

Comment:
I like this rule, but agree that it's not worth failing the build over.

Reply:
Scott haf tthe issue th at some tool he has auto fills in value=

I like the rule too and like a lot of these... Lcd. You could overide yourself but that's a consistency nightmare

Reply:
I guess the question... why does this rule matter?

Reply:
doesn't, just stylistic, looks nicer w/o `value`.